release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -74,6 +74,11 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 [id="ocp-4-12-networking"]
 === Networking
 
+[id="ocp-4-12-external-dns-operator"]
+=== External DNS Operator
+
+In {product-title} 4.12, the External DNS Operator modifies the format of the ExternalDNS wildcard TXT records on AzureDNS. The External DNS Operator replaces the asterisk with `any` in ExternalDNS wildcard TXT records. You must avoid the ExternalDNS wildcard A and CNAME records starting with `any` because this might cause a conflict.
```

Comment:
```suggestion
In {product-title} 4.12, the External DNS Operator modifies the format of the ExternalDNS wildcard TXT records on AzureDNS. The External DNS Operator replaces the asterisk with `any` in ExternalDNS wildcard TXT records. You must avoid the ExternalDNS wildcard A and CNAME records having `any` leftmost subdomain because this might cause a conflict.
```

Reply:
Acked